tests/ttnn/unit_tests/operations/test_slice.py

```diff
@@ -406,3 +406,45 @@ def test_slice_negative_ends(layout, dim, ends, device):
 
     ttnn_output = ttnn.to_torch(ttnn_output)
     assert_with_pcc(torch_output, ttnn_output, 0.99)
+
+
+@pytest.mark.parametrize(
+    "input_shape, input_start, input_ends",
+    (
+        ((1, 1, 1, 256), (0, 0, 0, 0), (1, 1, 1, -1)),
+        ((1, 256), (0, 0), (-1, 1)),
+        ((1, 512), (0, 0), (-1, 1)),
+        ((1, 512), (0, 0), (1, 256)),
+    ),
+)
+@pytest.mark.parametrize(
+    "layout",
+    (ttnn.TILE_LAYOUT, ttnn.ROW_MAJOR_LAYOUT),
+)
+def test_slice_bert(input_shape, input_start, input_ends, layout, device):
+    if layout == ttnn.TILE_LAYOUT:
+        torch_input = torch.randn(input_shape, dtype=torch.bfloat16)
+        ttnn_input = ttnn.from_torch(torch_input, device=device, dtype=ttnn.bfloat16, layout=layout)
+    else:
+        torch_input = torch.randn(input_shape, dtype=torch.float32)
+        ttnn_input = ttnn.from_torch(torch_input, device=device, dtype=ttnn.float32, layout=layout)
+
+    if len(input_shape) == 4:
+        torch_output = torch_input[
+            input_start[0] : input_ends[0],
+            input_start[1] : input_ends[1],
+            input_start[2] : input_ends[2],
+            input_start[3] : input_ends[3],
+        ]
+        ttnn_output = ttnn_input[
+            input_start[0] : input_ends[0],
+            input_start[1] : input_ends[1],
+            input_start[2] : input_ends[2],
+            input_start[3] : input_ends[3],
+        ]
+    elif len(input_shape) == 2:
+        torch_output = torch_input[input_start[0] : input_ends[0], input_start[1] : input_ends[1]]
+        ttnn_output = ttnn_input[input_start[0] : input_ends[0], input_start[1] : input_ends[1]]
```

Comment:
this is good, but can you please also test a call to `slice`?

Reply:
not supported on ttnn.slice since negative dims are not supported. We handle the wrap around on the Python side atm. I am making a change to move all that logic to ttnn.slice and make the __getitem__ overload a wrapper

Reply:
The second & third input from bert should be:
((1, 256), (0, 0), (-1, `256`)),
((1, 512), (0, 0), (-1, `512`))

Reply:
Whoops sorry!